.github/workflows/e2e.yml

```diff
@@ -22,13 +22,16 @@ jobs:
           path: |
             ~/.cargo/registry
             ~/.cargo/git
+            ~/.cargo/bin
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}-${{ matrix.rust }}-1
       - name: Install Rust
         run: |
           rustup update ${{ matrix.rust }} --no-self-update
           rustup default ${{ matrix.rust }}
           rustup component add rustfmt
+      - name: Install ic-cdk-optimizer
+        run: cargo install ic-cdk-optimizer
```

Comment:
This would fit better in the provisioning scripts, and then only for the rust tests.  Example: https://github.com/dfinity/sdk/blob/master/scripts/workflows/provision-linux.sh#L21
If installed here, ic-cdk-optimizer won't be available during the e2e tests, because those run in a later job on a different runner.

Reply:
Thanks!